w5e1.py

```diff
@@ -0,0 +1,82 @@
+"w5e1" 
+import random
+import os
+
+# pokemon s1e1 -> Ash late to professor Oak so can't choose pokemon...
+from datetime import datetime, timedelta
+
+MONDAYS = 1
+
+
+def pokemon():
+    pokemons = ['Bulbasaur', 'Squirtle', 'Charmander']
```

Comment:
Also, you can add it as a const outside

Reply:
Not sure I understood what that meant. Just change to capslock?